include/fmt/core.h

```diff
@@ -2169,6 +2169,7 @@ FMT_CONSTEXPR FMT_INLINE auto parse_arg_id(const Char* begin, const Char* end,
 template <typename Char, typename Handler>
 FMT_CONSTEXPR auto parse_width(const Char* begin, const Char* end,
                                Handler&& handler) -> const Char* {
+  using detail::auto_id;
```

Comment:
This should be unnecessary because we are in the detail namespace.

Reply:
My first impulse as well.
This is similar to #2260 and the outcome the same: the name-lookup of `detail::auto_id` fails with "`auto_id` is not a member of `detail`".

For more on that see below.

Reply:
I suggest moving namespace qualification to the only place where `auto_id` is used below.